### ISSUE TITLE
Add possibility to customize the container

### ISSUE THREAD

**Issue by AnnKolesnicova** [pradel/react-responsive-modal#41]
It could be named `node` as well in favour of https://github.com/pradel/react-minimalist-portal/issues/13

**Comment by Are0215** [comment]
Hi, Is there some news for this request?

**Comment by pradel** [comment]
@Are0215 you are right. It's implemented inside https://github.com/pradel/react-minimalist-portal/issues/13, should be pretty easy to add now, pull request welcome :)

**Comment by pradel** [comment]
published as `v3.1.0` see https://github.com/pradel/react-responsive-modal/releases/tag/v3.1.0 in order to know how to use it 🎉
